src/utils/folderutils.cpp

```diff
@@ -117,6 +118,23 @@ QDir FolderUtils::getAppPartsSubFolder(QString search) {
 
 QDir FolderUtils::getAppPartsSubFolder2(QString search) {
 	if (m_partsPath.isEmpty()) {
+#ifdef PKGDATADIR
+		QStringList candidates;
+		candidates.append(QDir::homePath() + "/.local/share/fritzing");
+		candidates.append("/usr/local/share/fritzing");
+		candidates.append(m_appPath);
+		foreach (const QString &candidate, candidates) {
+			QList<QDir> dirList;
+			dirList.append(QDir(candidate + "/fritzing-parts"));
+			dirList.append(QDir(candidate + "/parts"));
+			foreach (const QDir &dir, dirList) {
+				m_partsPath = dir.absolutePath();
+				if (dir.exists()) {
+					goto setpath;
+				}
+			}
+		}
+#else
```

Comment:
I think those guesses are not required, and are quite OS specific.
The --parts CLI option can be used instead.

Reply:
Yes these patches (and others at the time) were about getting 0.9.3b working on Linux. However, the paths are just those that are already in the code. This was just a way of rationalising their use - especially being able to avoid the directory recursing with 3724a283c88a5ed964ec6f23beddbb273ddbdda0.

I guess the other option is to remove all the code that is trying to find the parts in different locations, maybe leaving only applicationDirPath and PKGDATADIR (haven't thought about that too much).